### ISSUE TITLE
Dropdown Keyboard friendly  not work

### ISSUE THREAD

**Issue by viralchauhan** [CuppaLabs/angular2-multiselect-dropdown#471]
Dropdown is not work keyboard friendly.

**Comment by viralchauhan** [comment]
In List Up/Down Key not work

**Comment by dannetproject** [comment]
yes. u cant even use the keyboard to tap to the listitems.. for example.. if I press the key "t" it should goto the list items starting with the letter "t or T", but it doesnt.

**Comment by Ankit8010** [comment]
Can anyone have an idea, when will we have arrow key up/down functionality and selecting a particular list element by enter/space bar  ..??
